com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zcl/ZclCommand.java

```diff
@@ -122,10 +122,37 @@ public Integer getManufacturerCode() {
         return manufacturerCode;
     }
 
+    /**
+     * Sets the manufacturer code
+     *
+     * @param manufacturerCode the manufacturer code
+     */
+    protected void setManufacturerCode(int manufacturerCode) {
+        this.manufacturerCode = manufacturerCode;
+    }
+
+    /**
+     * When set to true, the default response message will not be generated
+     * 
+     * @return the disableDefaultResponse flag - when set to true, the default response message will not be generated
+     */
+    public boolean isDisableDefaultResponse() {
+        return disableDefaultResponse;
+    }
+
+    /**
+     * When set to true, the default response message will not be generated
+     *
+     * @param disableDefaultResponse true if the default response is not required
+     */
+    public void setDisableDefaultResponse(boolean disableDefaultResponse) {
+        this.disableDefaultResponse = disableDefaultResponse;
+    }
+
     @Override
     public String toString() {
         Integer resolvedClusterId = getClusterId();
-        final StringBuilder builder = new StringBuilder();
+        final StringBuilder builder = new StringBuilder(50);
```

Comment:
Whats the reason for artificially starting with a length of 50 here?

Reply:
It seemed like a reasonably number to cover the label etc. Otherwise I think it defaults to 16 and will most likely require a reallocation almost immediately. Maybe it could be reduced to 30 or 40 if you think it's too much?